openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -157,108 +186,146 @@
         },
 
         /**
-         Sends a request to the server to schedule the training of classifiers for
-         this problem's Example Based Assessments.
-
-         **/
+         * Sends a request to the server to schedule the training
+         * of classifiers for this problem's Example Based Assessments.
+         */
         scheduleTraining: function() {
             var view = this;
-            this.server.scheduleTraining().done(
-                function(msg) {
-                    $('#schedule_training_message', view.element).text(msg);
-                }
-            ).fail(function(errMsg) {
-                    $('#schedule_training_message', view.element).text(errMsg);
-                });
+            this.server.scheduleTraining()
+                .done(function(msg) {
+                    $('.schedule_training_message', view.element).text(msg);
+                }).fail(function(errMsg) {
+                $('.schedule_training_message', view.element).text(errMsg);
+            });
         },
 
         /**
-         Begins the process of rescheduling all unfinished grading tasks. This incdludes
-         checking if the classifiers have been created, and grading any unfinished
-         student submissions.
-
-         **/
+         * Begins the process of rescheduling all unfinished grading tasks.
+         * This includes checking if the classifiers have been created,
+         * and grading any unfinished student submissions.
+         */
         rescheduleUnfinishedTasks: function() {
             var view = this;
-            this.server.rescheduleUnfinishedTasks().done(
-                function(msg) {
-                    $('#reschedule_unfinished_tasks_message', view.element).text(msg);
-                }
-            ).fail(function(errMsg) {
-                    $('#reschedule_unfinished_tasks_message', view.element).text(errMsg);
-                });
+            this.server.rescheduleUnfinishedTasks()
+                .done(function(msg) {
+                    $('.reschedule_unfinished_tasks_message', view.element).text(msg);
+                }).fail(function(errMsg) {
+                $('.reschedule_unfinished_tasks_message', view.element).text(errMsg);
+            });
         },
 
         /**
-         Upon request, cancel the submission from grading pool.
-         **/
+         * Upon request, cancel the submission from grading pool.
+         */
         cancelSubmission: function(submissionUUID) {
             // Immediately disable the button to prevent multiple requests.
             this.cancelSubmissionEnabled(false);
             var view = this;
-            var sel = $('#openassessment__student-info', this.element);
-            var comments = sel.find('#staff-info__cancel-submission__comments').val();
-            this.server.cancelSubmission(submissionUUID, comments).done(
-                function(msg) {
+            var comments = this.element.find('.cancel_submission_comments').val();
```

Comment:
This line is throwing an error.

Uncaught TypeError: this.element.find is not a function